### PR TITLE
Fix kubemark support pre-existing

### LINKED ISSUE #91806: Kubemark pre-existing provider can not working
**What happened**:

Kubemark pre-existing provider can not work now, because of pre-existing code is a bit behind.

**What you expected to happen**:

Kubemark pre-existing provider can work well.

**How to reproduce it (as minimally and precisely as possible)**:

Follow the [kubemark pre-existing document](https://github.com/pytimer/kubernetes/blob/master/test/kubemark/pre-existing/README.md), use `test/kubemark/cloud-provider-config.sh` to create hollow-nodes.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
```sh
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.2", GitCommit:"59603c6e503c87169aea6106f57b9f242f64df89", GitTreeState:"clean", BuildDate:"2020-01-18T23:30:10Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.2", GitCommit:"59603c6e503c87169aea6106f57b9f242f64df89", GitTreeState:"clean", BuildDate:"2020-01-18T23:22:30Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
```sh
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```
- Kernel (e.g. `uname -a`):
`Linux master1 3.10.0-1062.18.1.el7.x86_64 #1 SMP Thu Apr 16 09:15:08 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux`
- Install tools: `kubeadm`
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by pytimer**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Kubemark  pre-existing provider  can works well with an existing kubemark master. 

**Which issue(s) this PR fixes**:

Fixes #91806 

**Special notes for your reviewer**:

I am not sure if remove 443 port in ` test/kubemark/resources/hollow-node_template.yaml` and `test/kubemark/resources/heapster_template.json` have affects other provider. If anyone know, please tell me. Thanks.

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by pytimer** [comment]
@mm4tt Some of tests failed, i am not sure how to deal with it, ignore or fix it, can you tell me? Thanks.

**Comment by mm4tt** [comment]
The `pull-kubernetes-verify` looks unrelated to your changes, let's try again

**Comment by mm4tt** [comment]
I have no idea why the test fails, it looks like it passes on other PRs- https://k8s-testgrid.appspot.com/presubmits-kubernetes-blocking#pull-kubernetes-verify
I'd reach out to test-infra folks for help

**Comment by dims** [comment]
it hits the upper limit for this CI job (2 hours) hence the failure.

/test pull-kubernetes-verify

**Comment by pytimer** [comment]
@mm4tt @dims 

I retest `pull-kubernetes-verify`, the failed results seems to timeout. 
The failed logs:

`{"component":"entrypoint","file":"prow/entrypoint/run.go:164","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 2h0m0s timeout","time":"2020-06-09T01:41:21Z"}`

`{"component":"entrypoint","file":"prow/entrypoint/run.go:245","func":"k8s.io/test-infra/prow/entrypoint.gracefullyTerminate","level":"error","msg":"Process did not exit before 15s grace period","time":"2020-06-09T01:41:36Z"}`

**Comment by pytimer** [comment]
All testing passed. Please review it, if something i missing or wrong, please tell me. Thanks.

File: cluster/pre-existing/util.sh [hunk 1]
```diff
@@ -54,3 +54,15 @@ function create-certs {
   export KUBELET_CERT_BASE64
   export KUBELET_KEY_BASE64
 }
+
+# Must ensure that the following ENV vars are set
+function detect-master {
+        if [[ -z "${MASTER_IP:-}" ]]; then
```
[the remark this answers is not in the document]
Comment (pytimer) [reply cluster/pre-existing/util.sh]: Done

**Pull request closed without merging**